EIPS/eip-template.md

```diff
@@ -0,0 +1,375 @@
+---
+eip: 
```

Comment:
```suggestion
eip: 7719
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.